lib/src/jobs_client.dart

```diff
@@ -0,0 +1,94 @@
+part of 'mistralai_client_dart_base.dart';
+
+/// Class responsible for managing jobs related operations.
+class JobsClient {
+  JobsClient._(
+    this._httpClient,
+    this._apiKey,
+    this._timeout,
+    this._apiUrlFactory,
+  );
+
+  /// HTTP client used to send requests.
+  final http.Client _httpClient;
+
+  /// required to authenticate requests.
+  final String _apiKey;
+
+  /// maximum wait time for response.
+  final Duration _timeout;
+
+  /// factory used to create URLs for requests.
+  final MistralAIUrlFactory _apiUrlFactory;
+
+  /// Creates a new fine-tune job.
+  ///
+  /// [model] specifies the model to be fine-tuned.
+  /// The [trainingFiles] contains as a list of file IDs of previously
+  /// uploaded training files via [FilesClient.create]. An optional list of
+  /// [validationFiles] can be provided to improve the fine-tuning process.
+  /// The [hyperparameters] specify the fine-tuning parameters for the job.
+  /// The [suffix] is an optional suffix added to the fine-tuned model name.
+  /// The [integrations] is an optional list of integrations to enable for
+  /// the fine-tuning job.
```

Comment:
nit: a little bit better readability in the IDE

```suggestion
  /// Creates a new fine-tune job.
  ///
  /// [model] specifies the model to be fine-tuned.
  /// 
  /// The [trainingFiles] contains as a list of file IDs of previously
  /// uploaded training files via [FilesClient.create].
  /// 
  /// An optional list of [validationFiles] can be provided to improve
  /// the fine-tuning process.
  /// 
  /// The [hyperparameters] specify the fine-tuning parameters for the job.
  /// 
  /// The [suffix] is an optional suffix added to the fine-tuned model name.
  /// 
  /// The [integrations] is an optional list of integrations to enable for
  /// the fine-tuning job.
```

Reply:
https://github.com/nomtek/mistralai_client_dart/pull/59/commits/a69cce334c2a4bbbf8673bedfff5e6a59850fc45